### PR TITLE
Update simulator

### PR DESCRIPTION
**Pull request by egthomas**
This pull request updates the radar simulator originally developed by AJ and @pasha-ponomarenko (`make_sim`) in several ways:

1. Now uses standard RST documentation, command line help, and option error handling
2. Correctly writes `iqdat`-format output files when using `-iq` option
3. Allows user control over radar ID, beam number, and first range containing scatter via new command line options (more could be added in the future)

To test, try comparing the output of the simulator with various options on this branch to the `develop` branch (note that a few command line options may have been changed / renamed for clarity or consistency with other RST routines).

### CODE REVIEW

**Comment by ecbland** [comment]
@egthomas Would you mind adding a modification line underneath the license notice in `make_sim.c?` 

I've never used the data simulator before, but I will try it out this week.
@pasha-ponomarenko, would you have time to test/review as well?

**Comment by egthomas** [comment]
Just a couple of small fixes here, most notably addressing an error when trying to set `-srng` (the first gate with scatter) and `-n_good` (the number of gates with scatter).

File: codebase/superdarn/src.bin/tk/tool/make_sim.1.0/doc/make_sim.doc.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+<option><on>--help</on><od>print the help message and exit.</od>
+</option>
+<option><on>--version</on><od>print the RST version number and exit.</od>
+</option>
+<option><on>-katscan</on><od>use Kathrn McWilliams' 8-pulse sequence (default).</od>
+</option>
+<option><on>-tauscan</on><od>use Ray Greenwald's 13-pulse sequence.</od>
+</option>
+<option><on>-oldscan</on><od>use the old 7-pulse seqeuence.</od>
```
Comment (ecbland) [inline codebase/superdarn/src.bin/tk/tool/make_sim.1.0/doc/make_sim.doc.xml:18]:
```suggestion
<option><on>-oldscan</on><od>use the old 7-pulse sequence.</od>
```

File: codebase/superdarn/src.bin/tk/tool/make_sim.1.0/doc/make_sim.doc.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+</option>
+<option><on>-srng <ar>srng</ar></on><od>set the first range gate containing scatter to <ar>srng</ar> (default is 0).</od>
+</option>
+<option><on>-nave <ar>nave</ar></on><od>set the number of averages in the integration period to <ar>nave</ar> (default is 70/50/20 for oldscan/katscan/tauscan).</od>
+</option>
+<option><on>-amp <ar>amp</ar></on><od>set the average ACF amplitude to <ar>amp</ar> (default is 1).</od>
+</option>
+<option><on>-nocri</on><od>Remove cross-range interference from the ACFs (default is CRI on). WARNING: removing cross-range interference will make the raw samples unusable, since each range gate will have to be integrated separately.</od>
+</option>
+<option><on>-decay</on><od>Set the ACFs to have a decaying ampliutde by a factor of 1/(r^2).</od>
```
Comment (ecbland) [inline codebase/superdarn/src.bin/tk/tool/make_sim.1.0/doc/make_sim.doc.xml:50]:
```suggestion
<option><on>-decay</on><od>Set the ACFs to have a decaying amplitude by a factor of 1/(r^2).</od>
```

**Review by ecbland — COMMENTED** [review]
@egthomas I've had a play with this and it seems to work, including the new command line options you added. Since you've modified someone else's code, could you please add a one-line description of your modifications to `make_sim.c`?

```
Modifications:
          <first and last name>, <institution> <year-month-day of the modification> : <comment on the change (optional)> 
```

**Pull request merged**